platform-common-string/src/main/java/com/softicar/platform/common/string/CommonStringI18n.java

```diff
@@ -0,0 +1,15 @@
+package com.softicar.platform.common.string;
```

Comment:
:warning: **Note to reviewer:** :warning:

Start your review at `CsvTokenizer.java`